dataset/dictionary/characters-playable.json5

```
@@ -3,18 +3,284 @@
   // The Astral Express
   //
   {
-    en: "",
-    ja: "",
-    zhCN: "",
+    en: "Himeko",
+    ja: "姫子",
+    zhCN: "姬子",
+    pronunciationJa: "ひめこ",
+    notes: "",
+    notesZh: "",
+    tags: [ "character-playable", "astral-express" ],
+  },
+  {
+    en: "Welt Yang",
+    ja: "ヴェルト・ヨウ",
+    zhCN: "瓦尔特·杨",
+    pronunciationJa: "",
+    notes: "",
+    notesZh: "",
+    tags: [ "character-playable", "astral-express" ],
+  },
+  {
+    en: "Trailblazer",
+    ja: "開拓者なり",
```

なり should be removed.

```suggestion
    ja: "開拓者",
```